ci/builders/mac_clang_tidy.json

```diff
@@ -67,6 +76,23 @@
                 "osx_sdk"
             ],
             "tasks": [
+                {
+                    "name": "gn for host_debug",
```

Comment:
This is a bit crufty, but ensures that the symlinks referred to by the `compile_commands.json` are created as expected.

Reply:
Does this mean for Mac builds, we need to run `gn` config twice to make it right? One before the ninja build, and the other here after ninja?

Reply:
This is only needed because the test is a global test and we disabled the code to copy the prebuilts folder and symlinks to/from cas. If the test is running as part of a build, then the extra gn isn't needed.